lol

```diff
@@ -0,0 +1,172 @@
+adoc-api/akka/persistence/query/scaladsl/ReadJournal.html...
```

Comment:
Thanks for noticing @xuwei-k, removed https://github.com/akka/akka/commit/9b6b43335f5de6f1f3289ee8614d5d79c75a1bab 
(used this file to debug javadoc failure using `./scripts/find-javadoc-error.pl`)